cli/src/main/kotlin/OrtMain.kt

```diff
@@ -26,8 +26,7 @@ import com.github.ajalt.clikt.completion.completionOption
 import com.github.ajalt.clikt.core.CliktCommand
 import com.github.ajalt.clikt.core.context
 import com.github.ajalt.clikt.core.subcommands
-import com.github.ajalt.clikt.output.CliktHelpFormatter
-import com.github.ajalt.clikt.output.HelpFormatter
+import com.github.ajalt.clikt.output.MordantHelpFormatter
```

Comment:
commit: Is it necessary to do the  upgrade and the removal of the helper formatter in one commit?

Reply:
Not strictly. I've made more fine-granular commits now.